### DISCUSSION TITLE
New xdrip and miband5

### DISCUSSION THREAD

**Discussion by rimjas** [NightscoutFoundation/xDrip#2059] [Q&A]
hey everyone! I have install new version 20220326 xdrip. How to enter miband 5 Auth code? Event list showed "Cannot authorize miband, pleasure recheck Auth code"

**Comment by parapenT1sta** [comment]
Because only MiBand 4 is supported in the xDrip Official release... You should use this version: https://www.patreon.com/posts/61121600
